README.md

```diff
@@ -1,9 +1,71 @@
 # terraform-scripts
 
 This repository holds our common Terraform Open edX deployment scripts.
+They can be used to provision and manage the infrastructure required by an Open edX instance on AWS.
 
-Currently it actually contains two different ways to provision AWS resources
+### TODOS:
+- [ ] Write usage documentation
+- [ ] Expand the conventions section below with more details and examples
+
+## Architecture map
+
+See [this document](https://docs.google.com/drawings/d/1s2hd6hJSKo1a-eqp6rI2kF-afACZXlK3f3ymptZjV6M/edit?usp=sharing).
+
+## Conventions
+### Resource naming
+
+Every resource provisioned by Terraform should be created using the following naming schema:
+
+```
+`{client_shortname}-{environment}-{resource}-{additional_info}`
```

Comment:
as discussed I think we should add an identifier for all resources, so that they can be easily identified to be something used by the openedx instance. In the modules I think I've used a combination of `edx-*` and `*-edxapp-*` but I guess this is still up for discussion (maybe we should mention opencraft instead?)

Reply:
@eLRuLL I went with a more generic approach and replaced `client_shortname` for `prefix` since it's essentially the same.
If you want the extra level of naming you can always set `prefix` as `edxapp-clientX`.
I think this is a cleaner approach, let me know what you think.

Reply:
sure makes sense :+1: